src/rgb/standards.rs

```diff
@@ -0,0 +1,53 @@
+//!Various RGB standards.
+
+use num::Float;
+use Yxy;
+use flt;
+use white_point::{WhitePoint, D65};
+use rgb::{Primaries, RgbSpace, RgbStandard};
+use pixel::TransferFn;
+
+///The sRGB color space.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct Srgb;
+
+impl Primaries for Srgb {
```

Comment:
I'm thinking of the situation where the user decides to invent their own RGB space by combining the primaries from sRGB with an other white point. It's not sRGB, in the same sense that AdobeRGB isn't sRGB (even if they share two primaries and the white point).

>These are the same right? What is the difference.

No, because the primaries are defined as xy values, which, as you know, is only perceptively affected by the white point. You would only adapt them to make them _look_ the same under a different light. This is a case where that's not the goal. I'm still not sure if this is a meaningful discussion and I should really read that link before I spiral away somewhere.